File: years/2022/clojure/aoc-2022/test/aoc_2022/day_10_test.clj
(ns aoc-2022.day-10-test
  (:require [aoc-2022.day-10 :refer :all]
            [clojure.test :refer :all]))

(deftest day-10-test
  (testing "sample data"
    (is (= 13140 (part1 "../../data/10-sample.txt"))))
;;     (is (= 1 (part2 "../../data/10-sample.txt")))
  (testing "puzzle data"
    (is (= 14160 (part1 "../../data/10.txt")))))
;;     (is (= 2445 (part2 "../../data/10.txt")))))

File: years/2024/elixir/test/aoc/day_10_test.exs
defmodule AoC.Day10.Test do
  @moduledoc false

  use ExUnit.Case, async: false

  alias AoC.Day10

  # comment this out to always log to the console
  @moduletag :capture_log

  describe "part 1" do
    test "with sample data" do
      assert Day10.part_1(["../data/10.txt"]) == 1
    end
  end

  describe "part 2" do
    test "with sample data" do
      assert Day10.part_2(["../data/10.txt"]) == 1
    end
  end
end
